deepsearch/__init__.py

```diff
@@ -0,0 +1,20 @@
+from notebook.utils import url_path_join
+from .handler import DeepsearchHandler
+
+
+def load_jupyter_server_extension(nb_app):
```

Comment:
That's the way he did it in quickopen and i liked that.
That way you import deepsearch instead of deepsearch.deepsearch
But I don't care moving it :)